src/kubernetes/reflector.rs

```diff
@@ -1070,7 +1071,7 @@ mod tests {
 
         // Prepare state.
         let (state_reader, state_writer) = evmap::new();
-        let state_writer = state::evmap::Writer::new(state_writer, None); // test without debounce to avouid complexity
+        let state_writer = state::evmap::Writer::new(state_writer, None, HashKey::Uid); // test without debounce to avouid complexity
```

Comment:
```suggestion
        let state_writer = state::evmap::Writer::new(state_writer, None, HashKey::Uid); // test without debounce to avoid complexity
```